### ISSUE TITLE
vgg16 and high loss and mAP is zero

### ISSUE THREAD

**Issue by nosa999** [fizyr/keras-retinanet#965]
I add a code to improve data augmentation and i run it with resnet50 backbone and run it correctly, but when I run it with vgg16 backbone it has high losses and mAP is zero

![image](https://user-images.githubusercontent.com/40929193/55712448-03b3e100-5a04-11e9-9218-ce7ea688d092.png)
 I run it in 40 epoch and it is like above image

**Comment by hgaiser** [comment]
The loss in the first epoch is very high, what are you using as a base? Are you starting from imagenet? You could try training for longer.

Regardless, there is no issue with our code here so I'll close this. Please use the Slack channel for usage questions.
